app/services/code_climate/projects_summary_service.rb

```diff
@@ -56,8 +56,8 @@ def metrics_in_given_technologies
     end
 
     def metrics_in_time_period
-      if from
-        metrics_in_department.where('snapshot_time >= ?', from)
+      if from && !from.zero?
```

Comment:
```suggestion
      if from.positive?
```

`from` will always be present (because fo the `.to_i`) and be zero or positive, right?

Reply:
right!

Reply:
i will change it in project detail service too